ecmascript/codegen/src/lib.rs

```diff
@@ -835,6 +835,7 @@ impl<'a> Emitter<'a> {
     #[emitter]
     fn emit_class_trailing(&mut self, node: &Class) -> Result {
         if node.super_class.is_some() {
+            space!();
```

Comment:
I got my approach was wrong. This updated code will insert space between `class ident` and `extends keyword`, but when minify option is false, space already exists after `class ident`, so when minify option is false, two spaces are inserted between  `class ident` and `extends keyword`.  I will fix this.